### DISCUSSION TITLE
Add support for recording and viewing of standard error output

### DISCUSSION THREAD

**Discussion by Cryolite** [smly/mjai.app#9] [Ideas]
It would be very helpful for debugging bots if there is a function to record and view the standard error output from the bots. At the very least, I would like to see the stack trace when the bots abort on an uncaught exception.

**Comment by smly** [comment]
Thank you very much for your helpful feedback! The implementation of this feature is complete. I plan to deploy it today or tomorrow.

**Reply by smly** [reply]
Released! https://github.com/smly/mjai.app/discussions/13

**Reply by Cryolite** [reply]
Great job! Thank you very much for taking out time of your precious spare time to implement it!
